R/dvhydrograph-render.R

```diff
@@ -74,7 +74,15 @@ createDVHydrographPlot <- function(reportObject){
   primarySeriesApprovals <- parsePrimarySeriesApprovals(reportObject, startDate, endDate)
   primarySeriesLegend <- fetchReportMetadataField(reportObject, 'primarySeriesLabel')
   approvals <- readApprovalBar(primarySeriesApprovals, timezone, legend_nm=primarySeriesLegend, snapToDayBoundaries=TRUE)
-  logAxis <- isLogged(priorityTS[['points']], priorityTS[['isVolumetricFlow']], excludeZeroNegativeFlag) && minMaxCanLog
+  primaryTSCanLog <- isLogged(priorityTS[['points']], priorityTS[['isVolumetricFlow']], excludeZeroNegativeFlag)
+  if (!isEmptyOrBlank(fieldVisitMeasurements)) {
+    fieldVisitMeasurementsCanLog <- isLogged(fieldVisitMeasurements, TRUE, excludeZeroNegativeFlag) 
+  }
+  else {
+    fieldVisitMeasurementsCanLog <- TRUE
+  }
```

Comment:
I think lines 78-83 should go inside of `parseFieldVisitMeasurements()` much like they do for `parseMinMaxIVs()`.